ngraph/python/src/ngraph/opset1/ops.py

```diff
@@ -3490,55 +2816,102 @@ def tensor_iterator(
+
+    :param data: The input tensor to be transposed
+    :param input_order: Permutation of axes to be applied to the input tensor
+    :return: Transpose node
+    """
+    return _get_node_factory_opset1().create("Transpose", as_nodes(data, input_order))
+
+
+def unsqueeze(data: NodeInput, axes: NodeInput, name: Optional[str] = None) -> Node:
+    """Perform unsqueeze operation on input tensor.
 
-    :param init_value:   Node producing a value to be returned instead of an unassigned variable.
-    :param variable_id:  Id of a variable to be read.
-    :param name:         Optional name for output node.
-    :return: ReadValue node
+    Insert single-dimensional entries to the shape of a tensor. Takes one required argument axes,
+    a list of dimensions that will be inserted.
+    Dimension indices in axes are as seen in the output tensor.
+
+    For example: Inputs: tensor with shape [3, 4, 5], axes=[0, 4]
+                 Result: tensor with shape [1, 3, 4, 5, 1]
+
+    :param data: The node with data tensor.
+    :param axes: List of non-negative integers, indicate the dimensions to be inserted.
+                  One of: input node or array.
+    :return: The new node performing an unsqueeze operation on input tensor.
     """
-    return _get_node_factory().create("ReadValue", [as_node(init_value)],
-                                      {"variable_id": variable_id})
+    return _get_node_factory_opset1().create("Unsqueeze", as_nodes(data, axes))
 
 
 @nameable_op
-def extract_image_patches(
-    image: NodeInput,
-    sizes: TensorShape,
-    strides: List[int],
-    rates: TensorShape,
-    auto_pad: str,
-    name: Optional[str] = None,
+def variadic_split(
+    data: NodeInput, axis: NodeInput, split_lengths: NodeInput, name: Optional[str] = None
 ) -> Node:
-    """Return a node which produces the ExtractImagePatches operation.
+    """Return a node which splits the input tensor into variadic length slices.
 
-    :param image:     4-D Input data to extract image patches.
-    :param sizes:     Patch size in the format of [size_rows, size_cols].
-    :param strides:   Patch movement stride in the format of [stride_rows, stride_cols]
-    :param rates:     Element seleciton rate for creating a patch.
-    :param auto_pad:  Padding type.
-    :param name:      Optional name for output node.
-    :return: ExtractImagePatches node
+    :param data: The input tensor to be split
+    :param axis: Axis along which the input data will be split
+    :param split_lengths: Sizes of the output tensors along the split axis
+    :return: VariadicSplit node
     """
-    return _get_node_factory().create(
-        "ExtractImagePatches",
-        [as_node(image)],
-        {"sizes": sizes, "strides": strides, "rates": rates, "auto_pad": auto_pad},
-    )
+    return _get_node_factory_opset1().create("VariadicSplit", as_nodes(data, axis, split_lengths))
+
+
+# Extend Node class to support binary operators
```

Comment:
Can we move this section to `ngraph/__init__.py`? This is executed at import-time and `opset1` could potentially not be imported. Circular imports could be an issue.

Reply:
Done